blockly/src/blockly/storeys/storeys.d.ts

```diff
@@ -7,14 +7,26 @@ declare class Minecraft {
 
   narrate(entity: string, text: string): void;
 
+  addRemoveItem(count: number, itemType: any);
+
   player(): any;
 }
 
+type EventType = "playerJoined";
+
+type MinecraftCallback = (m: Minecraft) => void;
+
 declare class Events {
-  whenCommand(command: string, callback: (m: Minecraft) => void): void;
+  whenCommand(command: string, callback: MinecraftCallback): void;
 
-  whenEntityRightClicked(entity: string, callback: (m: Minecraft) => void): void;
+  whenEntityRightClicked(
+    entity: string,
+    callback: (m: Minecraft) => void
+  ): void;
```

Comment:
@edewit reviewing this purely with a dumb "pattern matching", I was wondering if you intentionally reformatted `whenEntityRightClicked` instead of making it using that new `MinecraftCallback` that you introduced here? Would this work and be cleaner:

```suggestion
  whenEntityRightClicked(entity: string, callback: MinecraftCallback): void;
```